Frameworks/include/CGIWICBitmap.h

```diff
@@ -60,8 +62,15 @@ class CGIWICBitmapLock : public RuntimeClass<RuntimeClassFlags<RuntimeClassType:
         m_bytesPerRow = static_cast<size_t>(bpr);
     }
 
+    CGIWICBitmapLock(_In_ BYTE* data, _In_ const WICRect* region, _In_ size_t bytesPerRow, _In_ WICPixelFormatGUID pixelFormat)
+        : m_dataBuffer(data), m_pixelFormat(pixelFormat), m_locked_rect(region), m_bytesPerRow(bytesPerRow) {
+        m_texture = nullptr;
+    }
+
     ~CGIWICBitmapLock() {
-        m_texture->Unlock();
+        if (m_texture) {
+            m_texture->Unlock();
+        }
         m_texture = nullptr;
```

Comment:
This is not supposed to be thread safe, the design of the old DisplayTexture is confusing enough, this has been made much better via Jared's change (hopefully it'll migrate here soon).
until then this is a temporary fix.

---
In reply to: [88099416](https://github.com/Microsoft/WinObjC/pull/1371#discussion_r88099416) [](ancestors = 88099416)